spring-xd-runner/src/main/java/org/springframework/xd/dirt/modules/metadata/LogSinkOptionsMetadata.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.xd.dirt.modules.metadata;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import org.springframework.xd.module.options.spi.ModuleOption;
+import org.springframework.xd.module.options.spi.ModulePlaceholders;
+
+
+/**
+ * Captures options for the {@code log} sink module.
+ * 
+ * @author Eric Bottard
+ * @author Gary Russell
+ */
+public class LogSinkOptionsMetadata {
```

Comment:
would it be possible to include these optionsmetadata classes only within the sample modules? (either way, they will disappear once we have pure boot config property support in place)